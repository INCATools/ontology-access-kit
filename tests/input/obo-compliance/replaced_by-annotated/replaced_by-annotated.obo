format-version: 1.4
ontology: replaced_by-annotated

[Term]
id: X:1
is_obsolete: true
replaced_by: X:2 {source="PMID:123469"}

