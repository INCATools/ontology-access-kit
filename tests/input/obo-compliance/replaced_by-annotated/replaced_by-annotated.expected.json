{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/replaced_by-annotated.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/IAO_0100001",
      "lbl" : "term replaced by",
      "type" : "PROPERTY"
    }, {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://purl.obolibrary.org/obo/IAO_0100001",
          "val" : "X:2"
        } ],
        "deprecated" : true
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}